radicle/src/cob/issue.rs

```diff
@@ -49,7 +49,7 @@ pub enum CloseReason {
 }
 
 /// Issue state.
-#[derive(Debug, Default, Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Debug, Default, Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Serialize)]
 #[serde(rename_all = "camelCase", tag = "status")]
```

Comment:
So which is it? camelCase or kebab-case?

Reply:
It's camelCase, will take it into account, once I found a solution how to deserialize the tagged enum 🫡